dli_app/mod_auth/controllers.py

```diff
@@ -39,29 +39,10 @@
 # Set all routing for the module
 @mod_auth.route('/register/', methods=['GET', 'POST'])
 def register():
-    # TODO: Only allow registration from an approved email link
     form = RegistrationForm(request.form)
     if form.validate_on_submit():
```

Comment:
All of the validation logic was moved to the Form.validate() function, so we don't have to clutter the controllers.py files with validation logic any more.